File: Assignment_12/Q1.c
#include<stdio.h>

void DisplayDigit(int iNO)
{

    if (iNO < 0)
    {
        iNO = -iNO;
    }
  int iDigit = 0;
   
    while( iNO != 0)
    {
        iDigit = iNO % 10;
        printf("%d\n",iDigit);
        iNO = iNO / 10;
    }

}


int main()
{

    int iValue = 0;

    printf("Enter a number : \n");
    scanf("%d",&iValue);

    DisplayDigit(iValue);
    return 0;
}

File: Assignment_37/Q1.java
import java.util.Scanner;

class Bitwise
{
    public boolean ChkBit(int iNo, int iPos)
    {
        int iMask = 0x1;
        int iResult = 0;

        iMask = iMask << (iPos - 1);

        iResult = iNo & iMask;

        if(iResult == iMask)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

public class Q1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        Bitwise bobj = new Bitwise();

        int iValue = 0;
        int iPosition = 0;
        boolean bRet = false;

        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        System.out.println("Enter position : ");
        iPosition = sobj.nextInt();

        bRet = bobj.ChkBit(iValue, iPosition);

        if(bRet == true)
        {
            System.out.println("Result : TRUE");
        }
        else
        {
            System.out.println("Result : FALSE");
        }
        
        sobj.close();
    }
}
